config/initializers/omniauth.rb

```diff
@@ -1,12 +1,21 @@
 OmniAuth.config.logger = Rails.logger
 
+# Options for google oauth
+Orientation::Application.config.x.oauth_options = {
+ prompt: "select_account",
+ image_aspect_ratio: "square",
+ # we're displaying at 80 pixels, this is for high density ("Retina") displays
+ image_size: 160,
+ hd: ENV["ORIENTATION_EMAIL_WHITELIST"] || ""
+}
+
+# Other possible options when using an other oauth provider
```

Comment:
`Orientation::Application.config.x.oauth_options` needs to be a little more configurable if it's going to be used in this file below. I can think of a few possible solutions. In order from least favorable to most favorable IMO:

1. Make a flag in ENV that specifies which provider to use and have a case statement in this file
2. Allow `Orientation::Application.config.x.oauth_options` to be configured via YAML
3. Allow `Orientation::Application.config.x.oauth_options` to be configured via ENV. The environment variable could be parsed to a Ruby Hash

Thoughts @olivierlacan?